openstf/feature_engineering/feature_applicator.py

```diff
@@ -0,0 +1,128 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from abc import ABC, abstractmethod
+
+import numpy as np
+import pandas as pd
+
+from openstf.feature_engineering.apply_features import apply_features
+from openstf.feature_engineering.general import (
+    add_missing_feature_columns,
+    remove_extra_feature_columns,
+)
+
+LATENCY_CONFIG = {"APX": 24}  # A specific latency is part of a specific feature.
+
+
+class AbstractFeatureApplicator(ABC):
+    def __init__(self, horizons: list, feature_set_list: list = None) -> None:
+        """
```

Comment:
@JanMaartenvanDoorn Is seems that you use both `feature_set_list` and `feature_set_names` for the same thing? It would be more clear to use the same name for the same thing I think.

Reply:
well spotted, I thought I found them all. changed everything to feature_set_list